### PR TITLE
holesky ready cdvn

### PR DESCRIPTION
**Pull request by xenowits**
* Adds `holesky` as the default network in CDVN
* Replaces `geth` with `nethermind` as the default EL client

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,38 +5,34 @@ version: "3.8"
...
+      --JsonRpc.Enabled=true
+      --JsonRpc.JwtSecretFile="/root/jwt/jwt.hex"
+      --JsonRpc.EngineHost=0.0.0.0
+      --JsonRpc.EnginePort=8551
+      --JsonRpc.Host=0.0.0.0
+      --JsonRpc.Port=8545
+      --Sync.SnapSync=true
+      --HealthChecks.Enabled=true
+      --Metrics.Enabled=true
+      --Metrics.ExposePort=8008
```
Comment (pinebit) [inline docker-compose.yml:31]: 6060 is specified in prometheus.yml.

**Review by pinebit — CHANGES_REQUESTED** [review]
Wrong port?

File: docker-compose.yml (hunk 1 above)
Comment (xenowits) [reply docker-compose.yml:31]: @pinebit thanks, fixed!

**Review by pinebit — APPROVED** [review]

File: docker-compose.yml
[the remark this answers is not in the document]
Comment (xenowits) [reply docker-compose.yml:1]: @pinebit do you think we should add `depends_on:` for `charon` depending on `lighthouse`? which ones specifically you mean?

**Review by dB2510 — APPROVED** [review]

**Pull request merged**